src/main/resources/application-mysql.properties

```diff
@@ -1,6 +1,6 @@
 # database init, supports mysql too
 database=mysql
-spring.datasource.url=${MYSQL_URL:jdbc:mysql://localhost/petclinic}
+spring.datasource.url=jdbc:mysql://spring-petclinic-mydb-1:3306/petclinic
```

Comment:
Ideally we should avoid hardcoding specific endpoints into configuration files. In this particular case there is already a way to overwrite default value in original implementation.

Depending on situation and environment you might want to change this endpoint. New version of file will require code change to be merged and built. If we'll want to deploy multiple instances of application pointed to multiple mysql instances, then each of them will require a code change.

Related links:
- https://tldp.org/LDP/abs/html/parameter-substitution.html (bash syntax is a bit different, but here we have almost the same idea)
- https://12factor.net/config